charts/dataingestion-service/values-dataingestion-dev.yaml

```diff
@@ -0,0 +1,79 @@
+replicaCount: 1
```

Comment:
Somewhere in this values file you should reference the new variable "efsFileSystemId". As a default it should be blank and look like (line does not matter as long as it is the root value):

efsFileSystemId: ""